tools/skingen/generate_source.py

```
@@ -0,0 +1,951 @@
+        hd.line([(x, 0), (x, 7)], fill=(*col[:3], a))
+    put("Graphics/ScreenPlayDrums hit-bar.png", hit)
+
+    # Effects
+    put("Graphics/hit_fx.png", effect_burst(8, 32, CYAN, "ring"))
+    put("Graphics/ScreenPlayDrums chip wave.png", effect_burst(64, 64, CYAN, "wave"))
+    put("Graphics/7_WailingFlush.png", effect_burst(64, 64, MAGENTA, "radial"))
+    put("Graphics/7_Bonus.png", effect_burst(117, 88, CYAN, "star"))
+    put("Graphics/7_Bonus_100.png", word_plate(117, 88, "100", color=CYAN))
+    put("Graphics/7_explosion.png", effect_burst(256, 256, CYAN, "radial"))
+
+    # Wailing fire atlas — runtime expects 2688×720 (wide horizontal strip).
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)